docs/current_docs/manuals/developer/package-managers.mdx

```diff
@@ -0,0 +1,89 @@
+---
+slug: /manuals/developer/typescript/784253/package-manager
+displayed_sidebar: "current"
+toc_max_heading_level: 2
+title: "Alternative Package Managers"
+---
+
+# Alternative Package Managers
+
+:::important
+This page is only applicable to the TypeScript SDK. Alternative package managers are not currently supported in the Python and Go SDKs.
```

Comment:
AFAIK you can still use `poetry` and `pip` in Python, so does that impact this note @TomChv ?